CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
 
+* [#78](https://github.com/rubocop/rubocop-rails/pull/478): Fix `Rails/ReversibleMigrationMethodDefinition` cop's `Include`. ([@rhymes][])
```

Comment:
Can you update the PR number?

```suggestion
* [#478](https://github.com/rubocop/rubocop-rails/pull/478): Fix `Rails/ReversibleMigrationMethodDefinition` cop's `Include`. ([@rhymes][])
```

And can you add your account to the bottom of the changelog?

```diff
 [@ohbarye]: https://github.com/ohbarye
 [@tubaxenor]: https://github.com/tubaxenor
 [@olivierbuffon]: https://github.com/olivierbuffon
 [@leonp1991]: https://github.com/leonp1991
 [@drenmi]: https://github.com/drenmi
+[@rhymes]: https://github.com/rhymes
```